CHANGELOG.md

```diff
@@ -7,12 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED]
 
-## Added
+### Added
 
-- export `COVALENT_CONFIG_DIR=/tmp` inside sbatch script to enable filelock
```

Comment:
We shouldn't remove any lines from the changelog. In case we are reverting that change then lets mention it in the `Changed` section.